### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by jburel**
Update readme

### CODE REVIEW

File: linux/test/README.md [hunk 1]
```diff
@@ -5,19 +5,16 @@ This directory contains Dockerfiles for testing the installation walkthroughs.
...
+    docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_centos6_apache22
 
 See `docker run --help` for more information on these and other options
 for running docker images.
 
-Ubuntu 14.04 using apache needs the web configuration to be modified to
-enable the OMERO web server to run, see below.
-
-Centos 7 cannot be tested in this way as systemd doesn't fully work, see below.
+CentOS 7 cannot be tested in this way as systemd doesn't fully work, see below.
```
Comment (ximenesuk) [inline linux/test/README.md]: Since CentOS 6 ius now has further instructions below can this sentence be expanded to reflect that?

Comment (ximenesuk) [inline linux/test/README.md]: Also, while you are modifying this file could you change the couple of instances of systemd to `systemd`?

Comment (manics) [inline linux/test/README.md]: Systemd is the name of a suite of tools, not a command, though Wikipedia seems confused on the correct capitalisation https://en.wikipedia.org/wiki/Systemd

Comment (jburel) [inline linux/test/README.md]: @ximenesuk I was planning to do in the centos6 ius PR 
the same applies to the new parameter for webapp. 
but if you are happy to review the changes in this PR even if it covers the other two I am fine with that

Comment (ximenesuk) [inline linux/test/README.md]: @manics yes, I wasn't sure here whether it refers to the daemon or the bundle? Maybe use bold?

@jburel I'm fine to review the wider changes in the PR

**Comment by jburel** [comment]
@ximenesuk: I have more commits to come in the next few days so I removed the PR from standup list

File: linux/README.md [hunk 2]
```diff
@@ -11,11 +11,15 @@ Copy the files from this directory, then run one of the install scripts,
 	bash install_centos6_nginx.sh
 	bash install_centos6_py27_apache24.sh
 	bash install_centos6_py27_nginx.sh
+	bash install_centos6_py27_ius_apache22.sh
+	bash install_centos6_py27_ius_apache24.sh
+	bash install_centos6_py27_ius_nginx.sh
 	bash install_debian8_nginx.sh
```
Comment (dominikl) [inline linux/README.md:17]: The centos7 scripts could be listed, too, just to be complete.

Comment (dominikl) [inline linux/README.md:17]: Ah, forgot that CentOS 7 is a bit special and cannot not be installed fully automatic, so might be fine to not list it here explicitly.  But maybe worth adding a note about it, e. g. referring to the CentOS 7 remark in `linux/test/README.md`?

Comment (jburel) [inline linux/README.md:17]: CentOS 7 does not follow the same pattern, that's why it has not been listed.

**Comment by dominikl** [comment]
Ok, good to merge (my comment above is not a blocker) :+1:

**Pull request merged**